docs/plugins.md

```diff
@@ -13,50 +13,59 @@ The plugin system is still in **beta** for now.
 The plugins are used to extend the functionality of the wiki. Most of them are accessible through the use of `tags`.
 For now there are only a few supported.  
 
-- `[[draw]]` Allows you to add an **interactive drawio drawing** to the wiki.  
-- `[[info]]`, `[[warning]]`, `[[danger]]`, `[[success]]` Adds a nice **alert message**.
-- `[[ page: some-page ]]` Allows to show an other page in the current one.
-- `[[swagger link]]` Allows to insert a **swagger** block into the wiki page. Link in annotation should lead 
-  to a GET endpoint with .json openapi file. `[[swagger https://petstore3.swagger.io/api/v3/openapi.json]]` 
-  can be used as an example
-
-## Adding a plugin
-
-Add the plugin to the `plugins` folder and add the `foldername` to section `plugins` in the `wikmd-config.yaml` file.
-
-## Construction
-
-Plugins are listed inside the `plugins` folder. 
-
+### Drawio plugin
+Allows you to add an **interactive drawio drawing** to the wiki. Use `[[draw]]` 
+tag to insert a drawio block, that can be edited in page preview mode.
+
+### Alerts
+Allows to insert an alert message in the page text. Here is a list of 
+possible alert messages:
+- `[[info]]`
+- `[[warning]]`
+- `[[danger]]`
+- `[[success]]`
+
+### Embedded pages
+Allows to show another page in the current one.<br> Usage:<br>`[[page: some-page]]`<br> where `some-page`
+is the name of another page from the wiki
+
+### Swagger integration
+Allows to insert a **swagger** block into the wiki page. <br> Usage: <br> 
+`[[swagger link]]` 
+<br>
+where `link` is a link to a GET endpoint with .json openapi file.
+<br>
+`[[swagger https://petstore3.swagger.io/api/v3/openapi.json]]` can be used as an example.
+
+### Plantuml diagrams
+Allows to embed a plantuml diagram. 
+<br>Usage:<br>
 ```
-plugins/
-├─ plugin1/
-│  ├─ plugin1.py
-│  ├─ ...
-├─ plugin2/
-│  ├─ plugin2.py
-│  ├─ ...
-├─ .../
+```plantuml
+@startuml
+Alice -> Bob: Authentication Request
+Bob --> Alice: Authentication Response
+
+Alice -> Bob: Another authentication Request
+Alice <-- Bob: Another authentication Response
+@enduml
+'''
```

Comment:
Can you help me with this problem? Idk how to use apostrophes in the code block, they close this block, so i cant write a proper example. Made a temp fix, replaced apostrophes in the end of the example code block with ''', do you know how to make it look better?

Reply:
I also noticed the issue 😅 Could you put a ` \ ` before it?

Reply:
As it is inside the code block, escape characters dont work properly, here is an example:
```
Code here i start an example of a code block
```plantuml

CODE
CODE

here i want to show the end of a code block, 
if i use three apostrophes, then they close the code block,
and i cant esacape them, because this text is inside the code block and escape characters dont work here. 
\```

```

Reply:
I can make it without the code block, like this:
\```plantuml
@startuml
Alice -> Bob: Authentication Request
Bob --> Alice: Authentication Response
Alice -> Bob: Another authentication Request
Alice <-- Bob: Another authentication Response
@enduml
\```

Reply:
Apperently you can do it with 6 ` 
`````
```plantuml

CODE
CODE

here i want to show the end of a code block, 
if i use three apostrophes, then they close the code block,
and i cant esacape them, because this text is inside the code block and escape characters dont work here. 
```
``````

Reply:
Wow, thx) Fixed this problem